apps/frontend/src/pages/profile/profile.tsx

```diff
@@ -0,0 +1,39 @@
+import { Loader, PageLayout } from "~/libs/components/components.js";
+import { DataStatus } from "~/libs/enums/data-status.enum.js";
+import { useAppSelector } from "~/libs/hooks/hooks.js";
+
+import { EditUserForm } from "./components/components.js";
+import styles from "./styles.module.css";
+
+const Profile = (): JSX.Element => {
+	const { authenticatedUser, dataStatus } = useAppSelector(({ auth }) => auth);
+
+	const isLoading =
+		dataStatus === DataStatus.PENDING || dataStatus === DataStatus.IDLE;
+
+	return (
+		<PageLayout>
+			<div className={styles["profile-content"]}>
+				<h1 className={styles["label"]}>Profile</h1>
+
+				{isLoading ? (
+					<div className={styles["profile-loader"]}>
+						<Loader />
+					</div>
+				) : (
+					authenticatedUser && (
```

Comment:
It doesn't make sense to render the page if authenticatedUser is undefined/null. It's better to add
```ts
if (!authenticatedUser) {
  return <></>;
}
```
right after useAppSelector

BTW `ProtectedRoute` doesn't even allow non-authenticated user to visit this page